cldr-json/cldr-numbers-modern/main/bg/numbers.json

```diff
@@ -106,8 +106,8 @@
               "insertBetween": " "
             }
           },
-          "standard": "0.00 ¤",
-          "accounting": "0.00 ¤;(0.00 ¤)",
+	   "standard": "#,##0.00 ¤",
```

Comment:
Spaces were changed to a tab at the beginning of this line.

Reply:
Shouldn't this change be applied to the originating XML used to generate the json?   I don't think its intended to take PRs on the json itself.

Reply:
> Spaces were changed to a tab at the beginning of this line.

Done.

Reply:
> Shouldn't this change be applied to the originating XML used to generate the json? I don't think its intended to take PRs on the json itself.

Where I can find the XML files, so I can update those too? Thanks!

Reply:
The core project is https://github.com/unicode-org/cldr

The way to log a bug is documented here: https://cldr.unicode.org/index/bug-reports
The issue tracker is here: https://unicode-org.atlassian.net/jira/software/c/projects/CLDR/issues

Typically you would file a ticket proposing a PR. When accepted, send a PR to the core repo ensuring the ticket number is at the start of the commit message.

Reply:
> The core project is https://github.com/unicode-org/cldr
> 
> The way to log a bug is documented here: https://cldr.unicode.org/index/bug-reports The issue tracker is here: https://unicode-org.atlassian.net/jira/software/c/projects/CLDR/issues
> 
> Typically you would file a ticket proposing a PR. When accepted, send a PR to the core repo ensuring the ticket number is at the start of the commit message.

Thank you!